src/core/plugin_registry/grpc_plugin_registry.cc

```diff
@@ -109,6 +110,7 @@ void BuildCoreConfiguration(CoreConfiguration::Builder* builder) {
 #ifndef GRPC_NO_RLS
   RegisterRlsLbPolicy(builder);
 #endif  // !GRPC_NO_RLS
+  RegisterServerCallTracerFilter(builder);
```

Comment:
Is this positioning ensuring that this filter will be the last one in the stack, right before the connected_channel filter?

Reply:
(please view discussion on reviewable)